engine/access/subscription/streamer.go

```diff
@@ -89,7 +95,14 @@ func (s *Streamer) sendAllAvailable(ctx context.Context) error {
 		response, err := s.sub.Next(ctx)
 
 		if err != nil {
-			if errors.Is(err, storage.ErrNotFound) || errors.Is(err, storage.ErrHeightNotIndexed) || execution_data.IsBlobNotFoundError(err) {
+			if errors.Is(err, ErrResponseNotAvailableForBlock) {
```

Comment:
Consider getting rid of this and replacing it with handling a case where `response == nil && err == nil` .